skillful_network_server/src/main/java/fr/uca/cdr/skillful_network/services/impl/user/FollowStateTrackerServiceImpl.java

```diff
@@ -0,0 +1,104 @@
+package fr.uca.cdr.skillful_network.services.impl.user;
+
+import fr.uca.cdr.skillful_network.entities.user.Followable;
+import fr.uca.cdr.skillful_network.entities.user.Follower;
+import fr.uca.cdr.skillful_network.entities.user.Notification;
+import fr.uca.cdr.skillful_network.entities.user.User;
+import fr.uca.cdr.skillful_network.repositories.user.FollowStateTrackerRepository;
+import fr.uca.cdr.skillful_network.repositories.user.UserRepository;
+import fr.uca.cdr.skillful_network.services.user.FollowStateTrackerService;
+import fr.uca.cdr.skillful_network.services.user.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class FollowStateTrackerServiceImpl implements FollowStateTrackerService {
+
+    @Autowired
+    private FollowStateTrackerRepository fstRepository;
+
+    @Autowired
+    private UserRepository userRepository;
```

Comment:
Could we avoid to have two repositories here? And rely on the `UserService`?

Reply:
yes I will remove it later when I am sure I do not need it. 
In the mean time, I comment it